### ISSUE TITLE
Let's make a README.md

### ISSUE THREAD

**Issue by harrygfox** [Amigoat/Alpha#3]
@ldarby91 I would find it useful to have a set of instructions from cloning the repository through installing npm dependencies to running the server. Creating a document called README.md (or readme.md) with this information and adding it to the repository will mean it's content will be displayed on the home page of the repository.

Would you be able to make a start on some basic instructions and then we can add other details like our purpose and goals and beautify it later?

**Comment by ldarby91** [comment]
Yeah that sounds great I can get that started tomorrow. I need to find out some more about running a server on the cordova app so we can test it on the phone as running it locally isn't really going to cut it anymore. 

As soon as I find out the way we're going to do for the real thing I'll get writing stuff down. 
Have you managed to decide on a way to make a static home page that we can add stuff too or do you need more info before you start it?

**Comment by harrygfox** [comment]
Not yet. Exactly as you say, a little more info would be good. Being able to work locally with this repository would be my first step _(just to get my head around it a little bit)_. If I can play around with the code I'll be much more able to make a front page.

Is it just a case of cloning this and then running 
`$ node server.js` ?

**Comment by ldarby91** [comment]
Yeah I know what you mean.

Not quite, you'd need to install mongoDB and then create the same database but that is purely back end stuff. It's probably not worth playing around with that code because it's very likely to change after I've figured out how to connect to the remote database. 

I think the framework is probably quite important though, but saying that I don't know if we're doing it as a single page app like Angular or multiple like React I believe? Does that matter at this stage either?

Do you think we should stick to the layout we have before too? Or potentially switch the whole thing up? 

Also who's going to dress up as the goat first?

**Comment by harrygfox** [comment]
> I don't know if we're doing it as a single page app like Angular or multiple like React

I don't think it matters yet. Should we be forced to combine or modulate we can worry about it when the time comes. 

> Do you think we should stick to the layout we have before too? Or potentially switch the whole thing up?

I think we should stick with what we had before (except I'll code it much better this time) and at least that's a starting point on which to improve. .....`unless you have a new exciting idea?`
